### PR TITLE
Add PLECS 4.3.app v4.3.6

### PR DESCRIPTION
**Pull request by owiecc**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/plecs-standalone.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'plecs-standalone' do
+  version '4.3.6'
+  sha256 'a517cebe68283865b34107351958ae817da3e25baa0aebb704d587258ab32842'
+
+  url "https://www.plexim.com/sites/default/files/packages/plecs-standalone-#{version.dots_to_hyphens}_maci64.dmg"
```
Comment (vladimyr) [inline Casks/plecs-standalone.rb:5]:
```suggestion
  url "https://www.plexim.com/sites/default/files/packages/plecs-standalone-#{version.dots_to_hyphens}_maci64.dmg"
  appcast "https://www.plexim.com/download/latest_version?myversion=#{version}"
```

File: Casks/plecs-standalone.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  name 'PLECS'
+  homepage 'https://www.plexim.com/'
+
+  app 'PLECS 4.3.app'
```
Comment (vladimyr) [inline Casks/plecs-standalone.rb:9]:
```suggestion
  app "PLECS #{version.major_minor}.app"

  zap trash: [
               '~/Library/Application Support/Plexim',
               '~/Library/Preferences/com.plexim.PLECS.plist',
               '~/Library/Saved Application State/com.plexim.plecs.savedState',
             ]
```

File: Casks/plecs-standalone.rb (hunk 1 above)
Comment (owiecc) [reply Casks/plecs-standalone.rb:5]: May I ask how did you find the app cast link?

Comment (vladimyr) [reply Casks/plecs-standalone.rb:5]: I used a transparent network proxy and caught a request that was triggered through an in-app check for updates feature.

**Pull request merged**